sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -193,7 +230,28 @@ case class DropFunction(
     functionName: String,
     ifExists: Boolean,
     isTemp: Boolean)(sql: String)
-  extends NativeDDLCommand(sql) with Logging
+  extends NativeDDLCommand(sql) with Logging {
```

Comment:
Same here, this should not be `NativeDDLCommand` anymore. Then you don't need to pass in the `sql` string.